spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -1,8 +1,185 @@
+
+The prebuilt applications follow a naming convention too: `<functionality>-<type>-<binder>`. For example, `cassandra-sink-kafka` is a _Cassandra sink_ using the Kafka binder.
+
+=== Using the artifacts
+
+You either get access to the artifacts produced by Spring Cloud Stream Application Starters via Maven, Docker, or building the artifacts yourself.
+
+==== Maven and Docker accesss
+
+Starters are available as Maven artifacts in the [Spring repositories](). You can add them as dependencies to your application, as follows:
+
+[source,xml]
+```
+<dependency>
+  <group>org.springframework.cloud.stream.app</group>
+  <artifactId>spring-cloud-starter-stream-sink-cassandra</artifactId>
+  <version>1.0.0.BUILD-SNAPSHOT</version>
+</dependency>
+```
+
+From this, you can infer the coordinates for other starters found in this guide.
+While the version may vary, the group will always remain `org.springframework.cloud.stream.app` and the artifact id follows the naming convention `spring-cloud-starter-stream-<type>-<functionality>` described <<classification,previously>>.
+
+Prebuilt applications are available as Maven artifacts too.
+It is not encouraged to use them directly as dependencies, as starters should be used instead.
+Following the typical Maven `<group>:<artifactId>:<version>` convention, they can be referenced for example as:
+
+```
+org.springframework.cloud.stream.app:cassandra-sink-rabbit:1.0.0.BUILD-SNAPSHOT
+```
+
+Just as with the starters, you can infer the coordinates for other prebuilt applications found in the guide.
+The group will be always `org.springframework.cloud.stream.app`.
+The version may vary.
+The artifact id follows the format `<functionality>-<type>-<binder>` <<classification,previously described>>.
+
+The Docker versions of the applications are available in Docker Hub, at `https://hub.docker.com/r/springcloudstream/`. Naming and versioning follows the same general conventions as Maven, e.g.
+
+```
+docker pull springcloudstream/cassandra-sink-kafka
+```
+
+will pull the _latest_ Docker image of the _Cassandra sink_ with the Kafka binder.
+
+==== Building the artifacts
+
+You can also build the project and generate the artifacts (including the prebuilt applications) on your own.
+This is useful if you want to deploy the artifacts locally, for example for adding a new starter, or if you want to build the entire set of artifacts with a new binder.
+
+First, you need to generate the prebuilt applications.
+This is done by running the application generation Maven plugin.
+You can do so by simply invoking the corresponding script in the root of the project.
+
+```
+./generate.sh
+```
+
+The script will generate for the each of the prebuilt applications the following items:
```

Comment:
“For each of the prebuilt applications, the script will generate the following items:”